### PR TITLE
Documentation for grype integration tech preview

### PR DESCRIPTION
**Pull request by dspalmer99**
Engine docs updates for grype integration tech preview

### CODE REVIEW

**Comment by dspalmer99** [comment]
Also see the corresponding engine PR [here](https://github.com/anchore/anchore-engine/pull/1116.)

File: content/docs/grype/_index.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+linkTitle: "Access"
+weight: 1
+---
+
+Anchore Enterprise 3.1.0 includes a limited-functionality tech preview of an integration with [Grype](https://github.com/anchore/grype)
+for vulnerability scanning. This integration will replace the legacy vulnerability scanner in a future version of Anchore Enterprise,
+but is provided in this release in a preview capacity so users can try it out.
+
+***Note:*** This tech preview is not intended for use in production environments. It should be installed in sandbox exnvironments,
+and is strictly provided here so users can get start to use it and get a sense for the functionality. It may not include all of the
```
Comment (Toure) [inline content/docs/grype/_index.md:12]: I think this part of the sentence may read better as followed:
"so users can get start to use it and get a sense for the functionality" -> "to allow the user to gain an early preview into the upcoming functionality."

File: content/docs/grype/_index.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+functionality of the legacy scanner. Please report any issues found with it via a support ticket.
+
+### Installing
+As of 3.1.0, Anchore Enterprise can be configured to use either the legacy or grype vulnerability scanner. It is not possible to run
+both vulnerability scanners at the same time. This configuration is picked up at bootstrap, and cannot be changed ona running system. 
+
```
Comment (Toure) [inline content/docs/grype/_index.md:18]: I would place a note under this section to indicate that downgrading from grype mode to legacy is not supported and will cause irreversible problems.

Comment (Toure) [inline content/docs/grype/_index.md:18]: line 17 has a small typo:

"ona" -> "on a"

File: content/docs/overview/feeds/_index.md [hunk 3]
```diff
@@ -24,6 +24,16 @@ Anchore Enterprise Feeds has three high level components:
 * Database -- Stores the current state of the normalized data for serving via api
 * API -- Serves the data to clients, supporting update-only fetches.
 
+### Grype
+Anchore Enterprise 3.1 includes a limited-functionality tech preview of an integration with [Grype](https://github.com/anchore/grype) for vulnerability
+scanning. This integration will replace the legacy vulnerability scanner in a future version of Anchore Engine, but is provided here in a preview capacity
+so users can try it out.
+
+***Note:*** This tech preview is not intended for use in production environments. It should be installed in sandbox environments, and is strictly provided
+here so users can get start to use it and get a sense for the functionality. It may not include all the functionality of the legacy scanner. Please report
```
Comment (Toure) [inline content/docs/overview/feeds/_index.md:33]: I think this part of the sentence may read better as followed:
"so users can get start to use it and get a sense for the functionality" -> "to allow the user to gain an early preview into the upcoming functionality."

**Review by Toure — CHANGES_REQUESTED** [review]

**Comment by dspalmer99** [comment]
Thanks @Toure! I've addressed all of that feedback.

File: content/docs/grype/_index.md (hunk 2 above)
Comment (dspalmer99) [reply content/docs/grype/_index.md:24]: Done

File: content/docs/grype/_index.md (hunk 1 above)
Comment (dspalmer99) [reply content/docs/grype/_index.md]: I took your suggestion, but then refactored it a little further. I definitely agree it was too informal before, but I think it is more concise now.

File: content/docs/grype/_index.md (hunk 2 above)
Comment (dspalmer99) [reply content/docs/grype/_index.md:24]: Fixed.

**Review by Toure — APPROVED** [review]

**Pull request merged**